js/scripts.js

```diff
@@ -14,41 +14,42 @@ let pokemonRepository = (function(){
         return pokemonList;
       }
   
-      return {
+    /* created pokemon list with buttons as items displaying the name */
+      function addListItem(pokemon){
+        let pokemonList = document.querySelector('.pokemon-list');
```

Comment:
I couldn't find className with `.pokemon-list` hence an error being thrown. Let us add an HTML element in the `index.html` with the className.